components/Registry.tsx

```diff
@@ -180,6 +180,22 @@ const Registry = () => {
         />
         <div className="">
           <div className="max-w-screen-lg mx-auto px-4 sm:px-6 md:px-8 py-2 pb-8">
+            {name.startsWith("gh:") && (
+              <WarningMessage
+                title="deno.land/x/gh:owner:repo redirect deprecation notice"
+                body={`The https://deno.land/x/gh:owner:repo style redirects are deprecated and will be removed on August 1st 2020. Instead of importing via deno.land you can import this module directly from GitHub${
+                  raw ? ` via the URL ${sourceURL}` : ""
+                }.`}
+              />
+            )}
+            {name.startsWith("npm:") && (
+              <WarningMessage
+                title="deno.land/x/npm:project redirect deprecation notice"
+                body={`The https://deno.land/x/npm:project style redirects are deprecated and will be removed on August 1st 2020. Instead of importing via deno.land you can import this module directly from unpkg.com via the URL ${
```

Comment:
@lucacasonato AFAICT you probably wanted to remove the " via the URL " after unpkg.com here since it'll get repeated?

Reply:
Thanks. Fixed in #1302